zerver/tests/test_invite.py

```diff
@@ -2733,6 +2734,17 @@ def test_prereg_user_status(self) -> None:
         self.assertIsNotNone(user)
         self.assertEqual(user.delivery_email, email)
 
+    def test_prereg_user_object_has_full_name_set(self) -> None:
+        email = "Test <test1@zulip.com>"
```

Comment:
Would be good to test a couple of more formats like only email, or name including commas, etc.